packages/suite/src/views/guide/GuidePanel.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useEffect } from 'react';
+import styled from 'styled-components';
+
+import * as guideActions from '@suite-actions/guideActions';
+import { useSelector, useActions } from '@suite-hooks';
+import {
+    FeedbackTypeSelection,
+    Feedback,
+    GuideDefault,
+    GuideArticle,
+    GuideCategory,
+} from '@guide-views';
+
+import type { GuideCategoryType } from '@suite/types/suite/guide';
+
+const Wrapper = styled.div`
+    background: ${props => props.theme.BG_WHITE};
+    display: flex;
+    flex-direction: column;
+`;
+
+const ViewWrapper = styled.div`
+    height: 100%;
+    overflow-y: auto;
+`;
+
+const GuidePanel = (props: any) => {
```

Comment:
Please define explicit type. It would be just `className?: string;`, probably?

Reply:
Resolved.